packages/db/src/search/CollectionSearch.ts

```diff
@@ -7,20 +7,29 @@ import { BaseSearch } from './BaseSearch';
 import { DbQueryParameters, QueryEvent } from '../types/search';
 import { translatePostgresCollectionToApiCollection } from '../translate/collections';
 import { PostgresCollectionRecord } from '../types/collection';
+import { TableNames } from '../tables';
+import { BaseRecord } from '../types/base';
 
 const log = new Logger({ sender: '@cumulus/db/CollectionSearch' });
 
-/**
- * There is no need to declare an ApiCollectionRecord type since
- * CollectionRecord contains all the same fields from the api
- */
+type StatusTypes = 'running' | 'queued' | 'completed' | 'failed' | 'total';
```

Comment:
GranuleStatus is defined packages/types/api/granules.d.ts, you can extend that type.